### ISSUE TITLE
Refactor to convert object literal to multiple lines / one line

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/TypeScript#25771]
**TypeScript Version:**  3.1.0-dev.20180717

**Code**

```ts
const o = { a: 0, b: 1, c: 2 };
```

**Expected behavior:**

A refactor exists to convert to:
```ts
const o = {
    a: 0,
    b: 1,
    c: 2,
};
```
and back.

**Actual behavior:**

No such refactor. I have to do a lot of manual editing of whitespace.

**Comment by mhegazy** [comment]
I do not think you need a refactor. i would argue that hitting an enter anywhere inside the object literal should do the trick. in other words, every property should be on its own line unless the whole object literal is on the same line.

**Comment by andy-ms** [comment]
> what about type literal

Presumably the same.

**Comment by ajafff** [comment]
+ import and export specifiers
